src/tiles/tile-dispatcher.ts

```diff
@@ -5,6 +5,7 @@ import {
   SharedResizeObserverInterface,
   SharedResizeObserverResizeHandlerInterface,
 } from '@internetarchive/shared-resize-observer';
+import { SortParam } from '@internetarchive/search-service/dist/src/search-params';
```

Comment:
You should be able to import this just from `@internetarchive/search-service` without the full path. If you can't you may need to run `yarn` or rebase from `initial_work` to get the latest version of `search-service`

Reply:
Done. 

@jbuckner: note fuller path used in `app-root.ts` and `collection-browser.ts`